src/software/ai/hl/stp/play/shoot_or_chip_play_test.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+import pytest
+
+import software.python_bindings as tbots
+from proto.play_pb2 import Play, PlayName
+from software.simulated_tests.ball_enters_region import *
+from software.simulated_tests.simulated_test_fixture import (
+    simulated_test_runner,
+    pytest_main,
+    TestType,
+)
+from proto.message_translation.tbots_protobuf import create_world_state
+from proto.ssl_gc_common_pb2 import Team
+from proto.geometry_pb2 import Point, Vector, Angle, AngularVelocity
+
+from proto.ssl_gc_geometry_pb2 import Vector2
+
+
+def test_shoot_or_chip_play(simulated_test_runner):
+    def setup(arg):
```

Comment:
so the setup function is called with each param, so if we define setup as taking in no arguments this would cause an error. I'll replace it with *args to make it more clear.